### PR TITLE
Fix: typos.

### PR DESCRIPTION
**Pull request by athoune**
Fix some strange typos.

### CODE REVIEW

**Comment by dqminh** [comment]
@athoune build failed on DCO validation.

File: apparmor/gen.go [hunk 1]
```diff
@@ -82,13 +82,13 @@ func generateProfile(out io.Writer) error {
...
 // check if the tunables/global exist
-func tuntablesExists() bool {
+func tunablesExists() bool {
 	_, err := os.Stat("/etc/apparmor.d/tunables/global")
 	return err == nil
 }
 
 // check if abstractions/base exist
-func abstrctionsEsists() bool {
+func abstrctionsExists() bool {
```
Comment (dqminh) [inline apparmor/gen.go]: s/abstrctionsExists/abstractionsExists

File: apparmor/gen.go [hunk 2]
```diff
@@ -67,12 +67,12 @@ func generateProfile(out io.Writer) error {
...
 		Name: "docker-default",
 	}
-	if tuntablesExists() {
+	if tunablesExists() {
 		data.Imports = append(data.Imports, "#include <tunables/global>")
 	} else {
 		data.Imports = append(data.Imports, "@{PROC}=/proc/")
 	}
-	if abstrctionsEsists() {
+	if abstrctionsExists() {
```
Comment (dqminh) [inline apparmor/gen.go]: s/abstrctionsExists/abstractionsExists

**Comment by Mashimiao** [comment]
You haven't sign your commit.

**Comment by hqhq** [comment]
Why Gordon turtle robot is not here? :)

**Comment by jfrazelle** [comment]
hes only on the docker/docker repo, we are working on adding him to others
so he can be our turtle hero

**Comment by athoune** [comment]
I don't understand your jokes with turtles.

The patch is now signed.

**Comment by mrunalp** [comment]
@athoune Could you address the comments that @dqminh added?

**Comment by athoune** [comment]
aaaaaah, a "a" was missing. Fixed.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**